cms/djangoapps/contentstore/views/course.py

```diff
@@ -1367,12 +1389,21 @@ def group_configurations_list_handler(request, course_key_string):
         if 'text/html' in request.META.get('HTTP_ACCEPT', 'text/html'):
             group_configuration_url = reverse_course_url('group_configurations_list_handler', course_key)
             course_outline_url = reverse_course_url('course_handler', course_key)
-            configurations = GroupConfiguration.add_usage_info(course, store)
+            should_show_experiment_groups = are_content_experiments_enabled(course)
+            if should_show_experiment_groups:
+                experiment_group_configurations = GroupConfiguration.get_split_test_partitions_with_usage(course, store)
+            else:
+                experiment_group_configurations = None
+            content_group_configuration = GroupConfiguration.get_or_create_content_group_configuration(
+                course
+            ).to_json()
```

Comment:
It contains the JSON representation of a group configuration. The lengthy name was discussed in the PR review that this commit came from (this is our long-running branch, from which we have merged PRs that have been individually reviewed). We decided the name was OK as it does describe what the method does.

We won't be able to address nits on this PR, given our deadline.